src/components/EditActivityPage.js

```diff
@@ -31,12 +31,16 @@ const EditActivityPage = props => (
 );
 
 EditActivityPage.propTypes = {
-  activity: PropTypes.shape(activityPropTypeShape).isRequired,
+  activity: PropTypes.shape(activityPropTypeShape),
```

Comment:
I think, that this is wrong. I mean... If this is an `EditActivityPage`, activity has to exists... Isn't it true?